docs/release-notes/db-snapshot-flag.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  CLI: Add new `--db-snapshot` flag for the `det deploy aws up` subcommand that allows starting RDS
+   Database instances with a pre-existing snapshot. This flag is only currently usable with the
+   `simple-rds` deployment type.
```

Comment:
```suggestion
-  CLI: Add new ``--db-snapshot`` flag for the ``det deploy aws up`` subcommand that allows starting RDS
   DB instances with a pre-existing snapshot. This flag is only currently usable with the
   ``simple-rds`` deployment type.
```
(From a quick look at the RDS docs, it seems like they very consistently use "DB instance" for this.)